acceptance/bundle/deployment/bind/dashboard/output.txt

```diff
@@ -0,0 +1,37 @@
+
+=== Bind dashboard test: 
+=== Substitute variables in the template: 
+=== Create a pre-defined dashboard: 
+=== Bind dashboard: Updating deployment state...
+Successfully bound databricks_dashboard with an id '[DASHBOARD_ID]'. Run 'bundle deploy' to deploy changes to your workspace
+
+=== Deploy bundle: Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/bind-dashboard-test-[UUID]/default/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+=== Read the pre-defined dashboard: {
+  "display_name": "test dashboard [UUID]",
+  "lifecycle_state": "ACTIVE",
+  "path": "/Users/[USERNAME]/test dashboard [UUID].lvdash.json",
+  "parent_path": "/Users/[USERNAME]",
+  "serialized_dashboard": "{\"pages\":[{\"name\":\"02724bf2\",\"displayName\":\"Page One\"}]}"
+}
+
+=== Unbind the dashboard: Updating deployment state...
+
+=== Destroy the bundle: All files and directories at the following location will be deleted: /Workspace/Users/[USERNAME]/.bundle/bind-dashboard-test-[UUID]/default
+
+Deleting files...
+Destroy complete!
+
+=== Read the pre-defined dashboard again (expecting it still exists and is not deleted): {
+  "display_name": "test dashboard [UUID]",
+  "lifecycle_state": "ACTIVE",
+  "path": "/Users/[USERNAME]/test dashboard [UUID].lvdash.json",
+  "parent_path": "/Users/[USERNAME]",
+  "serialized_dashboard": "{\"pages\":[{\"name\":\"02724bf2\",\"displayName\":\"Page One\"}]}"
+}
+
+=== Test cleanup: 
+=== Delete the pre-defined dashboard [DASHBOARD_ID]: 0
```

Comment:
Not necessary to do now but one way to show how this works is to first commit a test (with errors in output.txt) and then a fix later with updated output.txt

Reply:
not sure what is an issue here? i will remove titles and exit code in a follow-up commit